### PR TITLE
Update Register Display Text

### PR DESCRIPTION
**Pull request by codepawpaw**

### CODE REVIEW

**Comment by codepawpaw** [comment]
Please review so it can merging faster

File: public/bin/index.bundle.js [hunk 1]
```diff
@@ -1047,6 +1017,36 @@ module.exports = warning;
...
+/***/ }),
+/* 10 */
+/***/ (function(module, exports, __webpack_require__) {
+
+"use strict";
+
+
+exports.__esModule = true;
+
+var _defineProperty = __webpack_require__(209);
```
Comment (bxcodec) [inline public/bin/index.bundle.js:1029]: what does this value do? `209`

File: public/bin/index.bundle.js [hunk 2]
```diff
@@ -3537,7 +3537,7 @@ var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol
 
 var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
 
-var _warning = __webpack_require__(10);
+var _warning = __webpack_require__(9);
```
Comment (bxcodec) [inline public/bin/index.bundle.js:3540]: what does this number do?

File: public/bin/index.bundle.js [hunk 3]
```diff
@@ -6774,7 +6774,7 @@ Object.defineProperty(exports, "__esModule", {
 });
 exports['default'] = createRule;
 
-var _warning = __webpack_require__(10);
+var _warning = __webpack_require__(9);
```
Comment (bxcodec) [inline public/bin/index.bundle.js:6777]: same here

**Review by bxcodec — CHANGES_REQUESTED** [review]
`public/bin/index.bundle.js.gz` 
Is this file generated?

Who will generate it?  If it generated by code or cli, and can be automated, please put this file into `.gitignore`

**Comment by bxcodec** [comment]
What really this things do ? :D 
Every number in webpack

**Comment by codepawpaw** [comment]
All file inside bin folder is generated by webpack. I already create pull request about that

**Comment by codepawpaw** [comment]
You can refer to another pr #8

**Review by bxcodec — APPROVED** [review]

**Comment by bxcodec** [comment]
LGTM

**Pull request merged**